script/deploy_ProdStack.s.sol

```diff
@@ -19,19 +19,20 @@ contract DeployProdStack is Script {
     function run() external {
         uint256 deployerPrivateKey = vm.envUint("PRIVATE_KEY");
         address settlement = vm.envAddress("SETTLEMENT");
+        bytes32 salt = vm.envBytes32("SALT");
 
         vm.startBroadcast(deployerPrivateKey);
 
         // Deploy ExtensibleFallbackHandler
-        new ExtensibleFallbackHandler{salt: "v1"}();
+        new ExtensibleFallbackHandler{salt: salt}();
```

Comment:
By having the salt hardcoded and checked into code, we can reproduce the builds if needed later (e.g. on other chains). If we pass it as an env, do we need to think about a way to document the used salt (or can it be easily recovered from the deployment tx)?

Reply:
Fair point. I'll revert this one. I was hoping that it was easy to recover from the broadcast JSON but I can't see it easily.